packages/ndla-ui/src/MessageBox/MessageBox.tsx

```diff
@@ -111,28 +136,36 @@ const CloseButtonText = styled.span`
     font-weight: ${fonts.weight.normal};
   }
 `;
+const LinkWrapper = styled.div`
+  display: block;
+  width: 100%;
+  background-color: #f9f4c8;
+  padding-bottom: 20px;
+  padding-left: 56px;
+`;
+const Link = styled.a`
+  color: blue;
+  font-size: 16px;
+  padding-bottom: 5px;
+  margin: 0px 50px 1px 5px;
+`;
 
 type Props = {
   type?: WrapperProps['boxType'];
   heading?: string;
   sticky?: boolean;
   onClose?: () => void;
-  children: React.ReactNode;
+  children?: React.ReactNode;
+  links?: []; //Takes and object with a name and a link. Eks: {'link1', 'www.link.no'}
 };
 
-export const MessageBox = ({
-  heading,
-  type = 'info',
-  sticky = true,
-  onClose,
-  children,
-  t,
-}: Props & WithTranslation) => (
+export const MessageBox = ({ heading, type, sticky = false, onClose, children, links, t }: Props & WithTranslation) => (
   <Sticky disabled={!sticky} stickyStyle={{ zIndex: 9999 }}>
     <Wrapper boxType={type}>
-      <InfoWrapper>
-        <IconWrapper>
-          <InformationOutline style={{ width: '24px', height: '24px' }} />
+      <InfoWrapper boxType={type}>
+        <IconWrapper boxType={type}>
+          {type == 'ghost' && <HumanMaleBoard style={{ width: '24px', height: '24px' }} />}
+          {type != 'ghost' && <InformationOutline style={{ width: '24px', height: '24px' }} />}
```

Comment:
Kva med 
```suggestion
          {type === 'ghost' ? <HumanMaleBoard style={{ width: '24px', height: '24px' }} /> : <InformationOutline style={{ width: '24px', height: '24px' }} />}
```

Reply:
Eventuellt definere stylinga i en const i og med at dei er like. Bruk forresten alltid === for likhetssjekk.
```suggestion
          {type === 'ghost' ? <HumanMaleBoard {...style} /> : <InformationOutline {...style} />}
```